nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -738,9 +738,13 @@ public Description matchParameterizedType(ParameterizedTypeTree tree, VisitorSta
     if (!withinAnnotatedCode(state)) {
       return Description.NO_MATCH;
     }
+
+    Symbol calledClass = castToNonNull(ASTHelpers.getSymbol(tree));
+    boolean isNullUnmarked = codeAnnotationInfo.isSymbolUnannotated(calledClass, config, handler);
+
```

Comment:
Let's move this code under the `if (config.isJSpecifyMode())` check below.

Also, it seems to me this logic could be simpler.  If `isNullUnmarked` is `true`, can't we just skip the call to `checkInstantiationForParameterizedTypedTree` entirely?  I don't see how that method would ever report an error if `isNullUnmarked` is `true`.  This way you could avoid changing `GenericsChecks`